### PR TITLE
Merge use package

### PR DESCRIPTION
**Pull request by steve-downey**
Land the use-package branch on master. 
PR created by creating fresh branch from master and merging the use-package branch to it, in order to resolve conflicts cleanly in `init.el` and `init-org.el`

### CODE REVIEW

File: modules/init-look-and-feel.el [hunk 1]
```diff
@@ -26,14 +26,14 @@
...
 (require 'init-prefs)
 
 

 ;;; Font
 
 (defvar exordium-available-preferred-fonts
-  (remove-if-not (lambda (font-and-size)
+  (cl-remove-if-not (lambda (font-and-size)
                    (member (car font-and-size) (font-family-list)))
```
Comment (pkryger) [inline modules/init-look-and-feel.el]: Consider re-align:

```suggestion
  (cl-remove-if-not (lambda (font-and-size)
                      (member (car font-and-size) (font-family-list)))
```

File: modules/init-ido.el [hunk 2]
```diff
@@ -5,7 +5,7 @@
 ;;; -------------- -------------------------------------------------------
 ;;; C-x C-r        Open recent file with IDO or Helm.
 
-(require 'ido)
+(use-package ido)
 (unless exordium-helm-everywhere
```
[the remark this answers is not in the document]
Comment (philippe-grenet) [reply modules/init-ido.el]: Yes that's better. FTR I also switched to Helm everywhere.

**Comment by steve-downey** [comment]
It's definitely worth cleaning this all up now, otherwise it will never get done.

File: modules/init-ido.el (hunk 2 above)
[the remark this answers is not in the document]
Comment (steve-downey) [reply modules/init-ido.el]: This does look better. I believe I discovered `:if` part way through.

[the remark this answers is not in the document]
Comment (steve-downey) [reply modules/init-ido.el]: Also, it seems this hasn't been exercised in a long time, as `ido-mode-both` is not a thing. This probably should be `ido-mode 'both`

**Comment by philippe-grenet** [comment]
@steve-downey do you think we can merge this today?

**Comment by steve-downey** [comment]
If I don't find any issues using it this morning, I will merge it today.

**Comment by philippe-grenet** [comment]
All right!!!

**Comment by philippe-grenet** [comment]
I cloned the repo from scratch and helm everywhere doesn't seem to be working. For example `M-x` used to show me a Helm command completion buffer, but it does not now. Will investigate tomorrow.

**Pull request merged**